src/main/java/com/google/devtools/build/lib/worker/WorkerFactory.java

```diff
@@ -77,9 +77,10 @@ public Worker create(WorkerKey key) throws Exception {
       reporter.handle(
           Event.info(
               String.format(
-                  "Created new %s %s worker (id %d), logging to %s",
+                  "Created new %s %s %s (id %d), logging to %s",
                   sandboxed ? "sandboxed" : "non-sandboxed",
                   key.getMnemonic(),
+                  WorkerKey.makeWorkerTypeName(key.getProxied()),
```

Comment:
I assume you meant the entire `WorkerKey.makeWorkerTypeName(key.getProxied())` statement, not just the `key.getProxied()`, since `key.getProxied()` three times in this functoin.

Let me know if you have more concerns. Thanks.